package.json

```diff
@@ -23,6 +24,14 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-minify": "^0.3.0",
     "backbone": "1.1.2",
+    "eslint": "^4.19.1",
+    "style-guide": "Invoca/style-guide#1804/m/STORY-2340_code_style_checker",
```

Comment:
This should probably be updated to point to just the repo name

Reply:
correct, i'll make the change